pkg/updater/resultstore/resultstore.go

```diff
@@ -379,6 +390,8 @@ func processGroup(tg *configpb.TestGroup, group *invocationGroup) *updater.Infla
 	hintTime := time.Unix(0, 0)
 	headers := make([][]string, len(tg.GetColumnHeader()))
 
+	receivers := make(chan updater.InflatedColumn)
```

Comment:
nit: Up to you, but I'd leave channels/parallelization for later. We can do correct behavior first and scale the processing up later once we know all the details work. (that should help the timeout in unit tests too).

Reply:
+1